Lmod/branches/7.0/z01-default_modules.sh

```diff
@@ -1,17 +1,17 @@
-if [[ -n "$USER_IS_ROOT" ]];then
-  if [ -z "$__Init_Default_Modules" ]; then
-     export __Init_Default_Modules=1;
-     if [ -z "$LMOD_SYSTEM_DEFAULT_MODULES" ];then
-       if [ -f "/usr/share/modulefiles/DefaultModules.lua" ];then
-	 export LMOD_SYSTEM_DEFAULT_MODULES="DefaultModules"
-       else
-	 if [ -f "/usr/share/modulefiles/StdEnv.lua" ];then
-	   export LMOD_SYSTEM_DEFAULT_MODULES="StdEnv"
-	 fi
-       fi
-     fi
-     module --initial_load restore
-  else
-     module refresh
+[ -z "$USER_IS_ROOT" ] && exit
+
+if [ -z "$__Init_Default_Modules" ]; then
+  export __Init_Default_Modules=1;
+  if [ -z "$LMOD_SYSTEM_DEFAULT_MODULES" ];then
+    if [ -f "/usr/share/modulefiles/DefaultModules.lua" ];then
+      export LMOD_SYSTEM_DEFAULT_MODULES="DefaultModules"
+    else
+      if [ -f "/usr/share/modulefiles/StdEnv.lua" ];then
+        export LMOD_SYSTEM_DEFAULT_MODULES="StdEnv"
+      fi
+    fi
   fi
+  module --initial_load restore
+else
+  module refresh
 fi
```

Comment:
yeah, there's a party of nested IFs over here; T.B.D. let's give it one more pass for now...